x-pack/plugins/security_solution/public/overview/components/detection_response/user_alerts_table/user_alerts_table.tsx

```diff
@@ -43,23 +43,30 @@ type GetTableColumns = (
 const DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID = 'vulnerableUsersBySeverityQuery';
 
 export const UserAlertsTable = React.memo(({ signalIndexName }: UserAlertsTableProps) => {
-  const { openTimelineWithFilters } = useNavigateToTimeline();
+  const openAlertsPageWithFilters = useNavigateToAlertsPageWithFilters();
 
-  const openUserInTimeline = useCallback(
-    ({ userName, severity }: { userName: string; severity?: string }) => {
-      const userNameFilter = { field: 'user.name', value: userName };
-      const severityFilter = severity
-        ? { field: 'kibana.alert.severity', value: severity }
-        : undefined;
+  const openUserInAlerts = useCallback(
+    ({ userName, severity }: { userName: string; severity?: string }) =>
+      openAlertsPageWithFilters([
+        {
+          title: 'Username',
+          selectedOptions: [userName],
+          fieldName: 'user.name',
+        },
 
-      openTimelineWithFilters(
-        severityFilter
-          ? [[userNameFilter, openAlertsFilter, severityFilter]]
-          : [[userNameFilter, openAlertsFilter]]
-      );
-    },
-    [openTimelineWithFilters]
+        ...(severity
+          ? [
+              {
+                title: 'Severity',
+                selectedOptions: [severity],
+                fieldName: 'kibana.alert.severity',
+              },
+            ]
+          : []),
```

Comment:
@jamster10 

[] array is not a valid value for a page filter type, so in this case if we want to select alerts of all severities then we should be adding severity page filter with selected options as empty array (`[]`) so that system knows that no value needs to be selected.

```suggestion
          : [ 
               {
                  title: 'Severity',
                  selectedOptions: [],
                  fieldName: 'kibana.alert.severity',
              }
             ]),
```